### PR TITLE
added some tests to isToday()

### PR DESCRIPTION
**Pull request by felipeiagocruz**
Hi! I'm interested in contributing to testing if no one has taken up this issue.
However, it is not clear to me what would be the correct way to submit these tests. For example in line 9 of the PR, PowerApps returns as false:
`>> IsToday(Date(-1, 2, 3))`
But the previous test said error ("#Error").
Can you check if I'm doing it right?

issue #283

### CODE REVIEW

File: src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/IsToday.txt [hunk 1]
```diff
@@ -6,7 +6,19 @@ true
...
 
 >> IsToday(Date(-1, 2, 3))
-#Error
+false
+
+>> IsToday(DateDiff(Now(),Today()))
+false
+
+>> IsToday("30,12,4")
+false
```
Comment (MikeStall) [inline src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/IsToday.txt:15]: This will be an error, passing the string to IsToday(). 

#Error(Kind=25)

**Comment by MikeStall** [comment]
An easy way to test and check for expected results would be to try the expressions inside of a Canvas app and see what the results are. There are corner cases around passing blank, passing "bad" inputs (like a string when a date is expected), etc.

**Comment by lesaltzm** [comment]
Make sure your canvas app has the error management experimental feature enabled, that should give you the correct results here.
image
image

**Comment by felipeiagocruz** [comment]
> Make sure your canvas app has the error management experimental feature enabled, that should give you the correct results here. image image

Thanks @lesaltzm! Now I'm getting the results back correctly, I'll start working on the issue.
